src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -24,8 +24,8 @@
 class SyliusApiExtension extends AbstractResourceExtension implements PrependExtensionInterface
 {
     protected $configFiles = array(
```

Comment:
@aRn0D Ok, but let's have an issue after merge to not forget about such change, ok?